src/figuresClasses.ts

```diff
@@ -1,19 +1,87 @@
-export interface Figure {
 
+function roundedArea(area:number):number {
```

Comment:
Great idea about name 🔥 
Just function names should start from the **verb**.
```suggestion
function roundArea(area:number):number {
```